### ISSUE TITLE
C-support: how to integrate c-support into existing vim config

### ISSUE THREAD

**Issue by shwsun** [WolfgangMehner/vim-plugins#40]
Hi,
I like this project a lot. I am just wondering can I use c-support as a vim plugin and add that to my existing vim config?

**Comment by WolfgangMehner** [comment]
You can use my repository [WolfgangMehner/c-support](https://github.com/WolfgangMehner/c-support) with a plug-in manager to install it. You can also download it and unpack it into your `$HOME/.vim` folder structure (`$HOME/.vim/ftplugin`, `$HOME/.vim/plugin`, ...).

**Comment by shwsun** [comment]
Right, I tried it (my config goes [here](git@github.com:shwsun/yavc.git)). The problem is that the keybinding wont work out of the box, I guess I will need to do some extra work or modify some setting right? Maybe it is just a problem with my own vim config.

**Comment by WolfgangMehner** [comment]
Are you referring to all the maps/hotkeys C-Support offers? ( https://github.com/WolfgangMehner/c-support/blob/3c35bc84314548a359682082bde1b7c057fbc8a8/doc/csupport.txt#L1195 )

I see you use the comma as a mapleader in your config. You can set it for C-Support by adding this line to your `.vimrc`:
> let g:C_MapLeader = ","

**Comment by shwsun** [comment]
@WolfgangMehner exactly what I want, thx.
One more thing, if people are using this project as the vim plugin, the Template file path will be something like `~/.vim/plugged/c-support/c-support/templates`

**Comment by WolfgangMehner** [comment]
Are you referring to the examples in the README, documentation, ... ?

These path are examples for the most basic installation using the archives from https://vim.sourceforge.io/scripts/script.php?script_id=213 . But all the path that are used internally by the plug-in are determined automatically and should work in any case.
